__tests__/user.test.js

```diff
@@ -0,0 +1,65 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const request = require("supertest");
+const app = require("../API/app");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("GET /api/users", () => {
+  describe("Functionality", () => {
+    test("Should return an array of User Objects", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body }) => {
+          const { users } = body;
+          expect(users.length).toEqual(4);
+          users.forEach((user) => {
+            expect(user.constructor).toEqual(Object);
```

Comment:
Interesting use of constructor. I think "instance of" would have worked as well, but I'm a little rusty on that.